### ISSUE TITLE
Selenium with its webdriver (chrome/firefox)

### ISSUE THREAD

**Issue by ricky-lim** [binder-project/binder#103]
I have been trying to install selenium with chrome/firefox webdriver for web-scraping, unfortunately it has not been successful with connection being refused.

Is there anyone who could give some pointers, with examples on the installation, preferably on the postBuild, please?

Thanks in advance.

Cheers

**Comment by ErenKaganNamal** [comment]
https://www.youtube.com/watch?v=7yYDOja8n_k
I wrote my first code thanks to this video. Try it

**Comment by choldgraf** [comment]
note that this project has moved! please open issues etc here: https://github.com/jupyterhub/binder/

also, please include a link to a URL for your repo, it makes debugging much easier

**Comment by ablekh** [comment]
@choldgraf Does https://mybinder.readthedocs.io cover the **new** Binder? Happy New Year to you, your family, your colleagues and all contributors to wonderful open source projects like Binder and Jupyter.

**Comment by choldgraf** [comment]
yes it does! those exist over at the `github.com/jupyterhub/` organization!
